src/components/content-item/Content-item.js

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import st from './content-item.module.scss';
+function ContentItem(props) {
```

Comment:
Можно так написать, если тебе все пропы не нужны
```suggestion
function ContentItem({children, header, time, hasExpand}) {
```

Reply:
принято